cice/Release-5.1/source/ice_therm_shared.F90

```diff
@@ -28,7 +28,7 @@ module ice_therm_shared
                          ! nilyr + 1 index is for bottom surface
 
       real (kind=dbl_kind), parameter, public :: &
-         ferrmax = 1.0e-3_dbl_kind    ! max allowed energy flux error (W m-2)
+         ferrmax = 9.0e-3_dbl_kind    ! max allowed energy flux error (W m-2)
```

Comment:
@caglartac we actually don't this change. The default ferrmax=1.0e-3_dbl_kind   is fine.